pkg/defaults/defaults.go

```diff
@@ -106,6 +106,11 @@ const (
 	// PidFilePath is the path to the pid file for the agent.
 	PidFilePath = RuntimePath + "/cilium.pid"
 
+	// XXX doc
```

Comment:
Please update (same for `DeleteQueueLockfile`).

The rest of the PR looks good to me, but I only took a very brief look at the Go code.

Reply:
ouch. good point.